backend/app/Services/Warehouse/WarehouseStockService.php

```
@@ -0,0 +1,106 @@
+        return $stock;
+    }
+
+    /**
+     * Current balance for a slot, 0 when the row does not exist yet — an untouched
+     * location holds nothing, which is the same answer as an emptied one.
+     *
+     * @param  array<string, int|null>  $keys
+     */
+    public function available(array $keys): float
+    {
+        return (float) (WarehouseStock::query()
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\WarehouseStock::query().


(staticMethod.notFound)